src/Virgil.vfj

```diff
@@ -5590,48 +5598,49 @@
-                        "362 497  444 515  496 520",
-                        "507 523  514 531  522 539",
-                        "528 565  519 586  490 590",
-                        "444 590",
-                        "385 596  320 577  262 571",
-                        "259 571  246 576  243 577",
-                        "231 589  237 661  230 678",
-                        "228 697  213 719  199 729",
-                        "175 733  159 730  142 710",
-                        "142 552",
-                        "136 545",
-                        "83 534  39 525  -19 523",
-                        "-51 492",
-                        "-60 481  -60 451  -54 437",
-                        "-40 426  -29 408  -13 403",
-                        "9 399  22 414  43 416",
-                        "74 432  101 436  142 438",
-                        "150 434  153 420  155 413",
-                        "155 401  161 339  161 324 s"
+                        "168 188",
+                        "168 187  168 187  169 186",
+                        "172 154  178 122  179 91",
+                        "172 42  205 7  221 -27",
+                        "238 -49  263 -56  289 -57 s",
+                        "307 -57  327 -45  343 -41",
+                        "455 -4  461 13  494 80",
+                        "506 98  506 120  509 141",
+                        "502 156  500 193  470 188",
+                        "445 189  437 177  430 158",
+                        "427 152  420 117  421 110",
+                        "389 56  381 59  336 48",
+                        "327 46  320 45  313 45 s",
+                        "297 46  288 54  277 64",
+                        "255 117  259 261  259 282",
+                        "248 327  255 364  255 408",
+                        "260 415",
+                        "277 421  299 427  319 428",
+                        "321 429",
+                        "374 440  423 454  475 458",
+                        "485 460  492 468  501 477",
+                        "507 502  498 523  469 527",
+                        "423 529",
+                        "365 536  334 521  276 516",
+                        "273 517  260 521  257 522",
+                        "244 535  252 607  245 623",
+                        "243 642  229 664  215 675",
+                        "191 680  175 676  158 657",
+                        "156 500",
+                        "149 492",
+                        "95 482  52 473  -6 472",
+                        "-39 443",
+                        "-48 431  -48 402  -43 388",
+                        "-29 376  -18 358  -2 352",
+                        "20 349  32 363  53 364",
+                        "85 380  112 383  154 386",
+                        "161 381  163 367  165 359",
+                        "166 347  170 285  170 270 s"
                       ]
                     }
                   ]
-                }
+                },
+                "active": true
```

Comment:
What does this `"active": true` stand for? Any idea?

Reply:
I have no clue, but the file still worked when I removed those attributes.